diff --git a/node_modules/react-native-webview/android/build.gradle b/node_modules/react-native-webview/android/build.gradle
index ae14145..b27988a 100644
--- a/node_modules/react-native-webview/android/build.gradle
+++ b/node_modules/react-native-webview/android/build.gradle
@@ -54,10 +54,12 @@ def supportsNamespace() {
 }
 
 apply plugin: 'com.android.library'
+if (!isNewArchitectureEnabled()) {
+    apply plugin: 'kotlin-android'
+}
 if (isNewArchitectureEnabled()) {
     apply plugin: 'com.facebook.react'
 }
-apply plugin: 'kotlin-android'
 
 android {
     if (supportsNamespace()) {
@@ -73,7 +75,7 @@ android {
         }
     }
 
-    compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
+    compileSdk getExtOrIntegerDefault('compileSdkVersion')
 
     defaultConfig {
         minSdkVersion getReactNativeWebViewMinSdkVersion()
